README.md

```diff
@@ -1,59 +1,62 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# BOOKSTORE
+
+> i made up Bookstore with reactjs 
+> i used state
+> i used setState
+> i used Redux
+> i used setState
+> Structure My application files using a "feature folder" approach and I used the ducks pattern for your Redux files.
 
-> One paragraph statement about the project.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5
+- CSS3
+- JavaScript
+- Reactjs
+- Redux
+- Api
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
+[Live Demo Link]
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to remove the **Live Demo Link** section if you have no demo link.

Reply:
I didn't yet complete my project because I need to add styling

Reply:
@AtillaTahak alright.